app/code/community/Reverb/ReverbSync/etc/config.xml

```diff
@@ -333,6 +333,12 @@
         <order_sync_reverb_source_url>/api/my/orders/selling/all?created_start_date=%s</order_sync_reverb_source_url>
         <!-- Default orders to be synced to the Admin store -->
         <store_to_sync_order_to>0</store_to_sync_order_to>
+        <!-- The set of order statuses which should be regarded as constituting NOT being a paid order -->
+        <non_paid_order_statuses>
```

Comment:
There are other statuses to take into consideration. https://dev.reverb.com/docs/syncing-orders#list-of-order-statuses .

I think we should have a white list instead of a black list. Basically, ignore everything that is not "paid".  All the other statuses mean it has already been shipped or the payment has not / did not go through.